components/Pages.php

```diff
@@ -706,15 +713,15 @@ public function edit_page_form() {
 		if ( $has_php ) {
 			pods_deprecated( 'Pod Page PHP code has been deprecated, please use WP Page Templates or hook into the pods_content filter instead of embedding PHP.', '2.1' );
 
-			if ( PODS_DISABLE_EVAL ) {
+			if ( defined( 'PODS_DISABLE_EVAL' ) && ! PODS_DISABLE_EVAL ) {
```

Comment:
Is this change correct? I still see an else statement that basically returns that same-ish message.
Maybe just remove the if/else statement and just return a single message?

Reply:
Good catch, I simplified these and also made it possible to use a filter to not show them (with that gist https://gist.github.com/sc0ttkclark/35a15e6f93622c45f12d671184fb1315)